### PR TITLE
Add & remove

### PR DESCRIPTION
**Pull request by Abel-desu**
**IN THIS PROJECT I USED :- **

- eventlistiner 
- add and remove functionality
- webpack
- and basic javascript

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,34 +1,151 @@
...
+    editInput.addEventListener('keypress', (e) => {
+      if (e.key === 'Enter') {
+        const todoContainers = document.querySelectorAll('.todoContainer');
+        const localData = JSON.parse(localStorage.getItem('list'));
+        for (let i = 0; i < todoContainers.length; i += 1) {
+          if (todoContainers[i].classList.contains('checkedContainer')) {
+            localData[i].description = editInput.value;
+            localStorage.setItem('list', JSON.stringify(localData));
+          }
+        }
```
Comment (youmari) [inline src/index.js]: - [ ] please make sure that you can edit a task right now I can't edit a task

File: src/index.js [hunk 2]
```diff
@@ -1,34 +1,151 @@
...
+};
+ /* eslint-disable */
+const removeTodo = (todo) => {
+  todosMaincontainer.removeChild(todo);
+  let count = 0;
+  const localData = JSON.parse(localStorage.getItem('list'));
+  const data = Array.from(localData).filter((i) => i.completed === false);
+  data.map((i) => i.index = count += 1);
+  localStorage.setItem('list', JSON.stringify(data));
+};
```
Comment (youmari) [inline src/index.js]: - [ ] please make sure  the task is removable at the moment when I click on the trash the task is not removed pleas try to fix that

File: src/index.js [hunk 3]
```diff
@@ -1,34 +1,151 @@
...
+  }
+});
+
+const getFromLocal = () => {
+  const data = JSON.parse(localStorage.getItem('list'));
+  data.map((i) => {
+    myArray.push(i);
     const todoContainer = document.createElement('div');
-    todoContainer.className = 'list border-bottom';
+    todoContainer.className = 'todoContainer';
```
Comment (youmari) [inline src/index.js]: - [ ] When making a task done `completed: true` you should save that to the local storage  right now when I check a task and reload the page it becomes unchecked again please try to fix that

**Review by youmari — CHANGES_REQUESTED** [review]
Hi @Abel-desu  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✔️ Great title and short summary 
✔️ No linter errors

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by FrederickMih — APPROVED** [review]
# Status: Approved! :tada: :tada: 

Hi @Abel-desu , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**